build/azure-pipelines.setup-go-workspace.sh

```diff
@@ -5,4 +5,4 @@ set -xeuo pipefail
 # Create GOPATH/bin and add it to our PATH so that
 # installed go binaries are available
 mkdir -p /home/vsts/go/bin/
-echo "##vso[task.prependpath]/home/vsts/go/bin/"
+echo "##vso[task.prependpath]/home/vsts/go/bin/"
```

Comment:
Is this file change significant?

Reply:
No seems like I removed the trailing new line, I can revert it?

Reply:
No that's fine, I just wasn't sure if we needed to make a similar change for other builds. 😀